jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/IO.java

```diff
@@ -452,27 +400,37 @@ public static boolean isEmptyDir(File dir)
      *
      * @param closeable the closeable to close
      */
-    public static void close(Closeable closeable)
+    public static void close(AutoCloseable closeable)
     {
         try
         {
             if (closeable != null)
                 closeable.close();
         }
-        catch (IOException ignore)
+        catch (Exception x)
         {
-            LOG.trace("IGNORED", ignore);
+            LOG.trace("IGNORED", x);
         }
     }
 
+    /**
+     * Closes an arbitrary closable, and logs exceptions at ignore level
+     *
+     * @param closeable the closeable to close
+     */
+    public static void close(Closeable closeable)
+    {
+        close((AutoCloseable)closeable);
+    }
```

Comment:
Cannot be removed?

Reply:
It should be able to be, but broke CI big time.
Jenkins is a mystery to me, so simplest to just leave it in:)